apps/src/applab/ViewDataButton.jsx

```diff
@@ -0,0 +1,49 @@
+/** @file Button that opens the data browser */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var styles = require('./PlaySpaceHeaderStyles');
+
+/**
+ * The button above the visualization that opens the data browser.
+ * @type {function}
+ */
+var ViewDataButton = React.createClass({
+  propTypes: {
+    onClick: React.PropTypes.func.isRequired
+  },
+
+  render: function () {
+
+    var showDataButtonStyle = $.extend(
+        {
```

Comment:
Is an argument list a line continuation?  I assumed it was, but I'm cool if we want to treat this like a block.